mix.exs

```diff
@@ -3,7 +3,7 @@ defmodule Ecto.Mixfile do
 
   @version "0.13.0-dev"
   @adapters [:pg, :mysql]
-  @pools [:poolboy]
+  @pools [:poolboy, :sojourn_broker]
```

Comment:
This means the integration tests will test each pool twice. Well, the unit tests too. I am not sure how to fix it, so you can let it be, I will think about some solutions.